requirements.txt

```diff
@@ -0,0 +1,8 @@
+# Required Packages w/ version
+astropy >= 3.2
+h5py >= 2.9
+matplotlib >= 3.0
+numpy >= 1.15
+scipy >= 1.0
+setuptools >= 41.2
+extension_helpers
```

Comment:
Like I said yesterday, `extension_helpers` seems mostly useless for us unless we're include C/Cython code.